opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -165,12 +165,13 @@ def to_metric(self, desc, tag_values, agg_data):
         metric_name = desc['name']
         metric_description = desc['documentation']
         label_keys = desc['labels']
+        tag_values = list(map(cast_none_to_empty_str, tag_values))
```

Comment:
Let's add a comment above this code to say
```python
# Prometheus requires that all tag values be strings hence
# the need to cast none to the empty string before exporting.
# See https://github.com/census-instrumentation/opencensus-python/issues/480
tag_values = list(map(cast_none_to_empty_str, tag_values)
```

Reply:
SG, done. Also fixed a bug in histogram buckets.

Reply:
Nit, but I think it's a bit more readable and idiomatic to do:

```python
tag_values = [tv for tv in tag_values if tv else ""]
```

Reply:
And why wrap the comment at <79 chars?

Reply:
Updated, thanks for the suggestion.

> And why wrap the comment at <79 chars?

Test session "lint" checks line length and will fail on lines with >80 chars, including comments.

Reply:
I mean why not wrap at the last word boundary before 79, the line is wrapped early. Not a big deal in any case.